libraries/helpers.rb

```diff
@@ -73,5 +73,17 @@ def home_basedir
     def pubkey_type(pubkey)
       %w(ed25519 ecdsa dss rsa dsa).filter { |kt| pubkey.split.first.include?(kt) }.first || 'rsa'
     end
+
+    # Returns a group for a new user to be added to
+    #
+    # @return [String, Integer]
+    def user_gid(user)
```

Comment:
I’m on mobile now so I’d need to look at it properly tomorrow but I’m 99% sure this can be refactored into something **much** nicer. The username var isn’t needed and there’s also the chance it’ll return nil (which may or may not be an issue).

Reply:
I removed the helper and inlined this, but I am not the best at shortening ruby code so lmk if there are further changes you would like.

Reply:
No, that's probably the way to go there.